src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -511,28 +510,26 @@ def update_rem_if_mgmt(self, if_oid, if_name):
             if len(mgmt_ip_str) == 0:
                 # the peer advertise an emtpy mgmt address
                 return
-            time_mark = int(lldp_kvs['lldp_rem_time_mark'])
-            remote_index = int(lldp_kvs['lldp_rem_index'])
-            subtype = self.get_subtype(mgmt_ip_str)
-            ip_hex = self.get_ip_hex(mgmt_ip_str, subtype)
-            if subtype == ManAddrConst.man_addr_subtype_ipv4:
-                addr_subtype_sub_oid = 4
-                mgmt_ip_sub_oid = (addr_subtype_sub_oid, *[int(i) for i in mgmt_ip_str.split('.')])
-            elif subtype == ManAddrConst.man_addr_subtype_ipv6:
-                addr_subtype_sub_oid = 6
-                mgmt_ip_sub_oid = (addr_subtype_sub_oid, *[int(i, 16) if i else 0 for i in mgmt_ip_str.split(':')])
-            else:
-                logger.warning("Invalid management IP {}".format(mgmt_ip_str))
-                return
-            self.if_range.append((time_mark,
-                                  if_oid,
-                                  remote_index,
-                                  subtype,
-                                  *mgmt_ip_sub_oid))
-
-            self.mgmt_ips.update({if_name: {"ip_str": mgmt_ip_str,
-                                            "addr_subtype": subtype,
-                                            "addr_hex": ip_hex}})
+            for mgmt_ip in set(mgmt_ip_str.split(',')):
+                time_mark = int(lldp_kvs['lldp_rem_time_mark'])
+                remote_index = int(lldp_kvs['lldp_rem_index'])
+                subtype = self.get_subtype(mgmt_ip)
+                if subtype == ManAddrConst.man_addr_subtype_ipv4:
+                    addr_subtype_sub_oid = 4
+                    mgmt_ip_sub_oid = (addr_subtype_sub_oid, *[int(i) for i in mgmt_ip.split('.')])
+                elif subtype == ManAddrConst.man_addr_subtype_ipv6:
+                    addr_subtype_sub_oid = 6
+                    mgmt_ip_sub_oid = (addr_subtype_sub_oid, *[int(i, 16) if i else 0 for i in mgmt_ip.split(':')])
```

Comment:
>[int(i, 16) if i else 0 for i in mgmt_ip.split(':')] [](start = 62, length = 52)

Seems you are trying to parse ipv6
```
[int(i, 16) if i else 0 for i in mgmt_ip.split(':')]
```
but this is not working for general cases

Reply:
@qiluo-msft Updated to use `exploded` API of ipaddress module.
https://docs.python.org/3/library/ipaddress.html